modules/ROOT/pages/install-checklist.adoc

```diff
@@ -30,6 +30,7 @@ Installing Anypoint Platform PCE requires specific versions of operating systems
 === Environment Prerequisites
 
 [%header,cols="1*a"]
+.Prerequisite Environment-Related Tasks
```

Comment:
The tables here and below result a bit confusing to me since we are first showing the Table name and number, and then the actual table. Also maybe add a title at the top saying "Prerequisites Tasks". Not sure if this is the standard tho. If so, nvm.

Reply:
Hey Ale. This is the syntax for the table title. How it is displayed is determined by the output HTML and CSS. It would be the same for all our docs. The question is whether we should be using table titles at all. ;-)